test/test_atacac/test_backup.py

```diff
@@ -0,0 +1,134 @@
+import os
+import shutil
+import textwrap
+from unittest import mock
+from test.utils import MockDir
+
+import yaml
+import pytest  # pylint: disable=import-error
```

Comment:
Why is pylint disabled here and in **all** other files with tests?

Reply:
`pylint` cannot handle this `import pytest` so I did an inline fix

```
************* Module test.test_atacac.test_validate
test/test_atacac/test_validate.py:7:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_backup
test/test_atacac/test_backup.py:7:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_synchronize
test/test_atacac/test_synchronize.py:6:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test___main__
test/test_atacac/test___main__.py:4:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_upload
test/test_atacac/test_upload.py:5:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_restore
test/test_atacac/test_restore.py:7:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_differentiate
test/test_atacac/test_differentiate.py:6:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test__utils
test/test_atacac/test__utils.py:7:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_validators.test___init__
test/test_atacac/test_validators/test___init__.py:6:0: E0401: Unable to import 'pytest' (import-error)
************* Module test.test_atacac.test_validators.test_builtin
test/test_atacac/test_validators/test_builtin.py:6:0: E0401: Unable to import 'pytest' (import-error)

```

Reply:
`pylint` can handle it and is showing valid warning about missing package `pytest`. You forgot to install it in tox env.

```diff
diff --git a/tox.ini b/tox.ini
index f856b20..6481021 100644
--- a/tox.ini
+++ b/tox.ini
@@ -30,7 +30,9 @@ commands = mypy --ignore-missing-imports {[VARS]sources}
 
 
 [testenv:pylint]
-deps = pylint>=2.5.3
+deps =
+    pylint>=2.5.3
+    pytest>=5.4.3
 commands = pylint -E {[VARS]sources}
```